pages/reference/supervisor/device-metrics.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Device Metrics
+---
+
+# Device Metrics
+
+The {{ $names.company.lower }} Supervisor reports metrics about your device to the cloud. These metrics are available on your balenaCloud dashboard, on the device summary page:
+
+![Device Metrics](https://www.balena.io/blog/content/images/2020/09/device-metrics-1.png)
+
+You can also get device metrics data through [our open API](https://www.balena.io/docs/reference/api/overview/), which allows you to port this data to a custom dashboard to monitor the utilization and health of your fleet. Note that device metrics are available with {{ $names.os.lower }} v2.56.0+rev1 and higher, running Supervisor v11.14.0 and higher.
```

Comment:
```suggestion
You can also fetch the device metrics data through balena's [open API](https://www.balena.io/docs/reference/api/overview/), [Node SDK](?) and [Python SDK](?). This allows you to port the metrics data to a custom dashboard in order to monitor the utilization and health of your fleet. Note that device metrics are available with {{ $names.os.lower }} v2.56.0+rev1 and higher, running Supervisor v11.14.0 and higher.
```

SDK could be mentioned as well?

Reply:
Changed with exception of:
- `You can also fetch the device metrics data` --> `You can also fetch device metrics data` (omit the)
- `This allows you to port the metrics data` --> `This allows you to port metrics data` (omit the)